src/lib/proxy-server.ts

```diff
@@ -209,10 +191,7 @@ export async function startProxyServer(): Promise< boolean > {
 
 		return true;
 	} catch ( error ) {
-		if (
-			( isErrnoException( error ) && error.code === 'EADDRINUSE' ) ||
-			( error instanceof Error && error.message === 'EADDRINUSE' )
-		) {
+		if ( error instanceof Error && error.message === 'PROXY_ERROR_PORT_IN_USE' ) {
 			throw new Error( 'PROXY_ERROR_PORT_IN_USE' );
```

Comment:
```suggestion
			throw error;
```
No need to construct a new error instance